### PR TITLE
Add a simple `WebSocketStream::send` method to replace `Sink` trait u…

### LINKED ISSUE #142: Provide a simple `async fn send` method for better interoperability with `futures-lite` based facilities
[Smol(`futures-lite`)'s guys think that `futures_sink::Sink` (till v0.3) is a legacy API and excluded related APIs in `futures-lite` intentionally. They prefer a simple `async fn send` API.](https://github.com/smol-rs/futures-lite/blob/5a355ae46a73c38a9a9b60c0940e983a325d799c/FEATURES.md#sink-trait)
Before a new simple `Sink` API is normalized, I suggest considering implementing a simple `async fn send` method:
```rust
impl<S> WebSocketStream<S> {
    /// Simple send method to replace `futures_sink::Sink` (till v0.3).
    pub async fn send(&mut self, msg: Message) -> Result<(), WsError>
    where
        S: AsyncRead + AsyncWrite + Unpin,
    {
        // Example to make it work. A simpler internal way can be used to implement it.
        SinkExt::send(self, msg).await
    }
}
```
In an ideal world, the `send` method would be called even if the `SinkExt` was introduced. I'm not sure if there would be additional problems if a method name is duplicated with a method in trait.
cc smol-rs/futures-lite#111 smol-rs/futures-lite#112

### PR DESCRIPTION
**Pull request by sdroege**
…sage

And also bump MSRV to 1.64.

Fixes https://github.com/sdroege/async-tungstenite/issues/142

### CODE REVIEW

**Comment by stackinspector** [comment]
LGTM

**Comment by sdroege** [comment]
I'll get a release out with this in the next days

File: src/lib.rs [hunk 1]
```diff
@@ -446,6 +448,84 @@ where
     }
 }
 
+impl<S> WebSocketStream<S> {
+    /// Simple send method to replace `futures_sink::Sink` (till v0.3).
```
Comment (Sytten) [inline src/lib.rs:452]: What does till v0.3 mean in this context?

Comment (stackinspector) [reply src/lib.rs:452]: A new simple Sink API maybe normalized in the future. See #142

**Pull request merged**